.github/workflows/unit-test-object.yml

```diff
@@ -0,0 +1,29 @@
+name: Unit Tests - Object Cache Service
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  tests:
+    name: "Object Service ${{ matrix.object }} - WP: ${{ matrix.wp_version }} - PHP: ${{ matrix.php }}"
```

Comment:
@imath Any option about removing the other `unit-test.yml` action? Since it performs the same task here with the only different being it uses wp-env. We'd keep the wp-env package, but not test it in Github.

Reply:
@imath  That's a good approach too. I updated the matrix with those suggestions. :)